loris/resolver.py

```diff
@@ -98,34 +106,9 @@ def resolve(self, ident):
         source_fp = self.source_file_path(ident)
         logger.debug('src image: %s' % (source_fp,))
 
-        format = self.format_from_ident(ident)
-        logger.debug('src format %s' % (format,))
-
-        return (source_fp, format)
-
-
-# To use this the resolver stanza of the config will have to have both the
-# src_img_root as required by the SimpleFSResolver and also an
-# [[extension_map]], which will be a hash mapping found extensions to the
-# extensions that loris wants to see, e.g.
-#
-# [resolver]
-# impl = 'loris.resolver.ExtensionNormalizingFSResolver'
-# src_img_root = '/cnfs-ro/iiif/production/medusa-root' # r--
-#   [[extension_map]]
-#   jpeg = 'jpg'
-#   tiff = 'tif'
-# Note that case normalization happens before looking up in the extension_map.
-class ExtensionNormalizingFSResolver(SimpleFSResolver):
```

Comment:
@jpstroop what do you think? You want to do deprecations for 2.2, and then remove stuff in 2.3? Note, though - we did already remove python 2.6 support after the 2.1 release without a deprecation notice (unless we revisit that decision).

Jon, looks like when you release you normally document the changes that users should be aware of - would it be helpful to have a document that we update whenever a change like this goes in?